src/vs/workbench/contrib/debug/browser/media/debug.contribution.css

```diff
@@ -87,6 +87,8 @@
 .monaco-workbench.mac .debug-pane .monaco-list-row .expression,
 .monaco-workbench.mac .debug-hover-widget .monaco-list-row .expression {
 	font-size: 11px;
+	display: flex;
+	align-items: center;
```

Comment:
`.expression` is already flex, and the `align-items` breaks a lot of other expressions

<img width="311" alt="image" src="https://user-images.githubusercontent.com/323878/208168343-0b3296d5-3140-4cc2-9c8c-6e93e5d7d994.png">

Reply:
Yes, it seems that there is such a problem...